setup.py

```diff
@@ -1,6 +1,16 @@
 import versioneer
 from setuptools import setup, find_packages
 
+with open('requirements.txt') as f:
+    requirements = f.read().split()
+
+requirements = [r for r in requirements if not r.startswith('git+')]
```

Comment:
Is this list comprehension for "requirements" necessary? You only want to get the requirements from GitHub to tell the user to install manually, right?

Reply:
It can only take care of packages not on `git+`. So that is why @tacaswell filter them out and print for the users so they are not surprised when the setup fails.
Issue #36 will eliminate the only dependency needed from `git+` from the project.

Reply:
It looks to me Tom overrode the `requirements` list with items that don't start with 'git+', and then look for items that start with 'git+' from the overwritten `requirements`.

@tacaswell, could you recheck this and line 8 below?

Reply:
ahh, that explains why it was not printing the list of installs!